src/firebaseConfig.js

```diff
@@ -0,0 +1,20 @@
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyDxON_6EsJlKUi5wX--JAiHV9i7G7WcDCI",
+  authDomain: "social-media-bed61.firebaseapp.com",
+  projectId: "social-media-bed61",
+  storageBucket: "social-media-bed61.appspot.com",
+  messagingSenderId: "284533140623",
+  appId: "1:284533140623:web:a84f35dc8127a0a78254fa",
```

Comment:
Do have a look into this and verify if the api key can be committed.

Reply:
Yes i will find this out and add to the .env, thanks!